dev-docs/bidders/raveltech.md

```diff
@@ -21,15 +21,38 @@ gvl_id: 32
+The RavelTech Prebid Adapter supports only Xandr/Appnexus SSP and should be configured with your current Xandr/Appnexus existing params:
+
+| Name                | Scope    | Description                                                                                                                                                                   | Example                                               | Type             |
+|---------------------|----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------|------------------|
+| `placement_id` (PBS+PBJS) or `placementId` (PBJS)    | required | The placement ID from AppNexus.  You may identify a placement using the `invCode` and `member` instead of a placement ID. This parameter can be either a `string` or `integer` for Prebid.js, however `integer` is preferred. Legacy code can retain the `string` value. **Prebid Server requires an integer value.**                                                    | `234234`                                            | `integer`         |
+| `member`                                        | optional | The member ID  from AppNexus. Must be used with `invCode`.                                                                                                                    | `'12345'`                                             | `string`         |
+| `invCode` or `inv_code`                             | optional | The inventory code from AppNexus. Must be used with `member`.                                                                                                                 | `'abc123'`                                            | `string`         |
+| `publisherId` or `publisher_id`                    | optional | The publisher ID from AppNexus. It is used by the AppNexus end point to identify the publisher when placement id is not provided and `invCode` goes wrong. The `publisherId` parameter can be either a `string` or `integer` for Prebid.js, however `integer` is preferred.                                                                                                                    | `12345`                                             | `integer`         |
+| `frameworks`                                    | optional | Array of integers listing API frameworks for Banner supported by the publisher. | `[1,2]` | `array of integer` |
+| `user`                                          | optional | Object that specifies information about an external user. See [User Object](#appnexus-user-object) for details.                                                               | `user: { age: 25, gender: 0, dnt: true}`              | `object`         |
```

Comment:
Please fix the #linkanchors on this line and 5 more below. Instead of #appnexus-blah, it needs to be /dev-docs/bidders/appnexus.html#appnexus-blah